packages/contracts-bedrock/test/DeploySuperchain.t.sol

```diff
@@ -12,84 +13,61 @@ import { DeploySuperchainInput, DeploySuperchain, DeploySuperchainOutput } from
 contract DeploySuperchainInput_Test is Test {
     DeploySuperchainInput dsi;
 
-    DeploySuperchainInput.Input input = DeploySuperchainInput.Input({
-        roles: DeploySuperchainInput.Roles({
-            proxyAdminOwner: makeAddr("defaultProxyAdminOwner"),
-            protocolVersionsOwner: makeAddr("defaultProtocolVersionsOwner"),
-            guardian: makeAddr("defaultGuardian")
-        }),
-        paused: false,
-        requiredProtocolVersion: ProtocolVersion.wrap(1),
-        recommendedProtocolVersion: ProtocolVersion.wrap(2)
-    });
+    address proxyAdminOwner = makeAddr("defaultProxyAdminOwner");
+    address protocolVersionsOwner = makeAddr("defaultProtocolVersionsOwner");
+    address guardian = makeAddr("defaultGuardian");
+    bool paused = false;
+    ProtocolVersion requiredProtocolVersion = ProtocolVersion.wrap(1);
+    ProtocolVersion recommendedProtocolVersion = ProtocolVersion.wrap(2);
 
     function setUp() public {
         dsi = new DeploySuperchainInput();
     }
 
-    function test_loadInput_succeeds() public {
-        // We avoid using a fuzz test here because we'd need to modify the inputs of multiple
-        // parameters to e.g. avoid the zero address. Therefore we hardcode a concrete test case
-        // which is simpler and still sufficient.
-        dsi.loadInput(input);
-        assertLoadInput();
-    }
-
     function test_loadInputFile_succeeds() public {
         string memory root = vm.projectRoot();
         string memory path = string.concat(root, "/test/fixtures/test-deploy-superchain-in.toml");
 
         dsi.loadInputFile(path);
-        assertLoadInput();
+
+        // Compare the test input struct to the getter methods.
```

Comment:
I think that this is a comment left over from the previous implementation.

Reply:
Fixed in bd36605f4